prediction_market_agent/agents/microchain_agent/nft_treasury_game/constants_nft_treasury_game.py

```diff
@@ -6,3 +6,5 @@
 NFT_TOKEN_FACTORY = Web3.to_checksum_address(
     "0x0D7C0Bd4169D090038c6F41CFd066958fe7619D0"
 )
+
+ENABLE_GET_MESSAGES_BY_HIGHEST_FEE = True
```

Comment:
Should we set this as ENV?
I would expect this to be set in a similar fashion as `ENABLE_NFT_TRANSFER` (https://github.com/gnosis/prediction-market-agent/blob/main/prediction_market_agent/agents/microchain_agent/microchain_agent_keys.py#L24)